website/docs/foundations/colors/partials/code/how-to-use.md

```diff
@@ -1,12 +1,16 @@
-## How to Use
+## How to use
 
 There are two different ways to apply a color to a UI element, via **design tokens** or via **CSS helper classes**.
 
-Which one to use will depend on the context: since both refer to the same design token values, from the design system perspective there's no difference between the two methods (it's more of a preference for the "consumer" codebase).
+Which one to use depends on the context. Since both refer to the same design token values, from the design system perspective there’s no difference between the two methods (it’s more of a preference for the “consumer” codebase).
```

Comment:
```suggestion
Which one to use depends on the context since both refer to the same design token values. From the design system perspective there’s no difference between the two methods (it’s more of a preference for the “consumer” codebase).
```